go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/stolostron/backplane-operator
 
-go 1.22.5
+go 1.23.4
```

Comment:
Is it best practice to update the go version in a previously delivered release?

Reply:
We're using 1.23.6 downstream so it's already updated.